dump-hints.py

```diff
@@ -1,58 +1,76 @@
-import gensim.models.keyedvectors as word2vec
-
-import heapq
-
-from tqdm import tqdm
+import io
+import pickle
 import re
 import time
+from zipfile import ZipFile
 
-import code, traceback, signal
+import gensim.models.keyedvectors as word2vec
+from tqdm import tqdm
 
+SIMPLE_WORD_REGEX = re.compile("^[a-z]*")
+DISALLOWED_ENDINGS = ['rlo', 'rlos', 'rla', 'rlas', 'rle', 'rles', 'ando', 'iendo', 'ía', 'ían', 'ísima', 'ísimo', 'ará', 'aré' \
```

Comment:
Add: "ías"

Reply:
Fixed.
This'll require regenerating the hints tables and deploying the db.